docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "System Table Integration"
+date: 2024-01-22T19:35:23+05:30
+---
+
+{{% notice note %}}
+At this time Overwatch only supports System Table Integration with AWS and Azure. 
+Currently System Table Integration is not supported on GCP.
+{{% /notice %}}
+
+Databricks system tables offer a robust mechanism for managing and understanding the intricacies of a Databricks 
+environment. From metadata exploration to performance optimization and security auditing, these tables provide a 
+comprehensive foundation for users and administrators to derive valuable insights and make informed decisions in a 
+collaborative and data-driven environment. Follow these documents to know more about System Tables 
+[Databricks Docs for System Tables](https://docs.databricks.com/en/administration-guide/system-tables/index.html)
+
+In our continuous efforts to enhance the efficiency and robustness of Overwatch, we have transitioned to a 
+system table integration for audit logs. This strategic move brings forth numerous advantages, further optimizing the 
+monitoring and logging capabilities of the tool. Currently, our integration focuses on leveraging system table 
+audit logs, but in future releases, we will be expanding our integration to include other system tables as well. Follow 
+these documents to know more about 
+[System Table Audit Logs](https://docs.databricks.com/en/administration-guide/system-tables/audit-logs.html)
+
+Pros of Moving to System Table Integrations: 
+
+* **Seamless Integration with Overwatch**:
+System table integration seamlessly aligns with Overwatch.
+The synergy between system tables and Overwatch ensures a harmonious integration, providing a unified platform for 
+comprehensive monitoring and analysis.
+
+* **Simplified Setup within Overwatch**:
+Integration with system tables leads to a simplified setup within Overwatch. The need to configure audit log settings 
+independently within the Overwatch configurations is eliminated. This simplicity ensures a more user-friendly experience
+for both setup and ongoing management. 
+
+* **Reduced Dependency on IAM Roles and Service Accounts**:
```

Comment:
We have mentioned this will be specifically for Audit Log management. If you want we can rename the heading to 
`Reduced Dependency on IAM Roles and Service Accounts for Audit Logs setup`

Reply:
I think we should still remove this, there's no real win with this point.

Reply:
Should we just merge these two points ?
* Simplified Setup within Overwatch
*Reduced Dependency on IAM Roles and Service Accounts